storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObjects.java

```diff
@@ -127,8 +129,42 @@ public Builder(@NonNull StorIOContentResolver storIOContentResolver, @NonNull It
          * @return builder
          */
         @NonNull
-        public Builder<T> withMapFunc(@NonNull MapFunc<T, DeleteQuery> mapFunc) {
+        public CompleteBuilder<T> withMapFunc(@NonNull MapFunc<T, DeleteQuery> mapFunc) {
             this.mapFunc = mapFunc;
+            return new CompleteBuilder<>(this);
+        }
+    }
+
+    /**
+     * Builder for {@link PreparedDeleteObjects}
+     *
+     * @param <T> type of objects
+     */
+    public static class CompleteBuilder<T> extends Builder<T> {
+
+        CompleteBuilder(@NonNull final Builder<T> builder) {
+            super(builder.storIOContentResolver, builder.objects);
+            deleteResolver = builder.deleteResolver;
+            mapFunc = builder.mapFunc;
+        }
+
+        /**
+         * {@inheritDoc}
+         */
+        @NonNull
+        @Override
+        public Builder<T> withDeleteResolver(@NonNull DeleteResolver deleteResolver) {
```

Comment:
I think, return type should be `CompleteBuilder<T>`? Because user will have to set `MapFunc` again

And probably, there are same problems in previous PRs, sorry, just thought about it now